cmd/list.go

```diff
@@ -29,6 +30,11 @@ var listCmd = &cobra.Command{
 	Long: `Gives a list of all the dependencies of the project. 
 	These include both direct as well as transitive dependencies.`,
 	RunE: func(cmd *cobra.Command, args []string) error {
+
+		if len(args) != 0 {
+			return errors.New("Invalid input! This command does not take any argumant as input.")
```

Comment:
I think we should go with `fmt.Errorf` because then we won't need to import another package here. Also please see #38 (comment)